project/src/main/java/com/google/sps/Servlets/CommentServlet.java

```diff
@@ -0,0 +1,30 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.*;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.Date;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/Comment")
+public class CommentServlet extends HttpServlet {
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```

Comment:
should this be a doPost method since you are putting data into the Database?

Reply:
+1, well probably switch this to post and then add a Get to retrieve the comment data for when we load the page

Reply:
yea max and I will work this one out today